### DISCUSSION TITLE
Add gammas and their energies in decay chain diagram

### DISCUSSION THREAD

**Discussion by tassanom** [radioactivedecay/radioactivedecay#79] [Ideas]
Thank you for the package, it was very good for me.

One suggestion...as the title says, I think it would be useful to include gamma emissions and their energies in the decay chain diagram.

Thanks
Marcos

**Comment by alexmalins** [comment]
Hey Marcos, Thanks for this suggestion. Yes, we are thinking of adding decay radiation data to this package at some point. Having an option to display the data on decay diagrams is a good idea that would be nice to have. We'd have to think about how to do it without the diagrams getting really complicated - maybe by limiting the number of steps in the decay chain shown...

Thanks again for the suggestion.

**Reply by alexmalins** [reply]
One way of keeping things clean would be to have an option for just displaying the primary gamma emissions on the decay chain diagrams. I.e. the gamma with the highest energy emission rate (energy * emission probability).

The option could be extended for displaying the primary beta, alpha emissions etc

**Reply by tassanom** [reply]
Yes, it is a challenge to condense all that information into a diagram. Perhaps, as you say, it would be a good start to include only the main energies (high probability) of all the decays.
